x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/BaseSearchableSnapshotIndexInput.java

```diff
@@ -101,34 +102,6 @@ public final void close() throws IOException {
 
     public abstract void innerClose() throws IOException;
 
-    protected InputStream openInputStream(final long position, final long length) throws IOException {
```

Comment:
Pushed down to the cached index input only, the other implementation doesn't use this.